app/src/main/java/com/example/notes2text/usecases/FileEditingUseCases/OpenTextEditorInteractor.java

```diff
@@ -0,0 +1,25 @@
+package com.example.notes2text.usecases.FileEditingUseCases;
+
+import java.io.File;
+
+public class OpenTextEditorInteractor implements OpenTextEditorBoundary {
+    File lastFile;
+
+    String lastFileRoot;
```

Comment:
Consider making these variables private, since there are getter functions for them